### PR TITLE
feat: Inform user there are unread messages in a conversation

### PR DESCRIPTION
**Pull request by przemvs**
Screen reader will annouce about unread messages, description, status.

### CODE REVIEW

File: src/script/components/list/ConversationListCell.tsx [hunk 1]
```diff
@@ -151,11 +151,58 @@ const ConversationListCell = ({
...
-    [Availability.Type.BUSY]: t('userAvailabilityBusy'),
-  };
+  const availabilityStrings: Record<string, string> = useMemo(
+    () => ({
+      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
+      [Availability.Type.AWAY]: t('userAvailabilityAway'),
+      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
+    }),
+    [],
+  );
```
Comment (atomrc) [inline src/script/components/list/ConversationListCell.tsx]:
```suggestion
  const availabilityStrings = useMemo(
    () => ({
      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
      [Availability.Type.AWAY]: t('userAvailabilityAway'),
      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
    } as const),
    [],
  );
```

That should work, right?

File: src/script/components/list/ConversationListCell.tsx [hunk 2]
```diff
@@ -182,7 +229,8 @@ const ConversationListCell = ({
           onKeyDown={handleDivKeyDown}
           data-uie-name="go-open-conversation"
           tabIndex={isFocused ? TabIndex.FOCUSABLE : TabIndex.UNFOCUSABLE}
-          aria-label={t('accessibility.openConversation', displayName)}
+          // aria-label={t('accessibility.openConversation', displayName)}
```
Comment (atomrc) [inline src/script/components/list/ConversationListCell.tsx]:
```suggestion
```

File: src/script/components/list/ConversationListCell.tsx [hunk 3]
```diff
@@ -151,11 +151,58 @@ const ConversationListCell = ({
...
+  const availabilityStrings: Record<string, string> = useMemo(
+    () => ({
+      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
+      [Availability.Type.AWAY]: t('userAvailabilityAway'),
+      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
+    }),
+    [],
+  );
+
+  const unreadMessages = unreadState.allMessages.length;
```
Comment (atomrc) [inline src/script/components/list/ConversationListCell.tsx]:
```suggestion
  const unreadMessagesCount = unreadState.allMessages.length;
```
To make it clear that it's a number

**Review by atomrc — APPROVED** [review]
A few suggestions, but looks good overall

File: src/script/components/list/ConversationListCell.tsx [hunk 4]
```diff
@@ -157,17 +157,65 @@ const ConversationListCell = ({
...
-  };
+  const availabilityStrings = useMemo(
+    () =>
+      new Map<Partial<Availability.Type>, string>([
+        [Availability.Type.AVAILABLE, t('userAvailabilityAvailable')],
+        [Availability.Type.AWAY, t('userAvailabilityAway')],
+        [Availability.Type.BUSY, t('userAvailabilityBusy')],
+      ]),
+    [],
+  );
```
Comment (V-Gira) [inline src/script/components/list/ConversationListCell.tsx:168]:
```suggestion  
  const availabilityStrings = useMemo(
    () => ({
      [Availability.Type.AVAILABLE]: t('userAvailabilityAvailable'),
      [Availability.Type.AWAY]: t('userAvailabilityAway'),
      [Availability.Type.BUSY]: t('userAvailabilityBusy'),
    } as const),
    [],
  );
```

The suggestion from @atomrc  is a bit different, not really qualified to comment tbh

Comment (przemvs) [reply src/script/components/list/ConversationListCell.tsx:168]: I change it to Map, with proper types :)

**Review by V-Gira — APPROVED** [review]

**Pull request closed without merging**